src/conf_mode/vpn_openconnect.py

```diff
@@ -45,7 +45,8 @@
 
 # Generate hash from user cleartext password
 def get_hash(password):
-    return crypt(password, mksalt(METHOD_SHA512))
+    return sha512_crypt.hash("password")
```

Comment:
Remove the quotes, this will always return the hash for string "password".

Reply:
Resolved, thank for your comment!